node/src/models/Mentor.ts

```diff
@@ -16,7 +16,7 @@ import {
   PaginateQuery,
   pluginPagination,
 } from './Paginatation';
-import { Answer, Status } from './Answer';
+import { Answer, mediaNeedsTransfer, Status } from './Answer';
```

Comment:
let's move that `mediaNeedsTransfer` function to a `utils/static-urls` module (see above)